app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.order(:name)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @recent_posts = Post.where(author_id: params[:id]).order(created_at: :desc).limit(3)
```

Comment:
- [ ] Great job so far :+1: Kindly use the `recent_posts` method you created in the user model to display the 3 most recents posts :100: 
- Requirements:
![Screenshot from 2022-08-17 16-48-41](https://user-images.githubusercontent.com/85113567/185184726-0cbfa230-7828-4184-8324-1d9a861d28df.png)